### DISCUSSION TITLE
Welcome to Open FPGA Stack

### DISCUSSION THREAD

**Discussion by ronyschutzInt** [OFS/ofs.github.io#1] [Announcements]
Welcome to the Open FPGA Stack Discussions Page! You can use discussions for:
- General questions regarding our collateral or the OFS architecture
- Sharing suggestions, ideas, or tips you have learned

Please post questions to our general discussion using our labels to identify your question. For possible bugs, please submit to our "Issues" tab.

In this community, all members can respond to questions, helping one another. Please remember open-source etiquette of being respectful and considerate in all discussions.

**Comment by pulak-gautam** [comment]
Currently OPAE / OFS uses PCIe interface to communicate with hardware (if I am not wrong). Is it possible to use an AXI instead of PCIe for the same? Further, is the data-mapping (in PCIe) detailed somewhere in the documentation, I was intrigued to understand how the data is being communicated particularly when OFS is integrated with OneAPI.
